tests/suites/test_suite_x509parse.function

```diff
@@ -219,6 +219,83 @@ int verify_print( void *data, mbedtls_x509_crt *crt, int certificate_depth, uint
 
     return( 0 );
 }
+
+int verify_parse_san( mbedtls_x509_subject_alternative_name *san,
+                      char **buf, size_t *size )
+{
+    int ret;
+    size_t i;
+    char *p = *buf;
+    size_t n = *size;
+
+    ret = mbedtls_snprintf( p, n, "type : %u", san->type );
+    MBEDTLS_X509_SAFE_SNPRINTF;
+
+    switch( san->type )
+    {
+       case( MBEDTLS_X509_SAN_OTHER_NAME ):
+       {
+           ret = mbedtls_snprintf( p, n, "\notherName :");
+           MBEDTLS_X509_SAFE_SNPRINTF;
+
+           if( MBEDTLS_OID_CMP( MBEDTLS_OID_ON_HW_MODULE_NAME,
```

Comment:
we wouldn't reach the else branch, because, if the o`othername` is not hardware module name, at the moment, `x509_get_other_name` inside `mbedtls_x509_parse_subject_alternative_name` would return `MBEDTLS_ERR_X509_FEATURE_UNAVAILABLE`. This is tested in `X509 SAN parsing, unsupported otherName name` test